src/components/NavbarStudy.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {FC, useState} from 'react';
+import './NavbarStudy.scss';
+import NavbarSVGSelector from './NavbarStudySVGSelector';
+
+const NavbarStudy: FC = () => {
+    const [isMenuActive, setIsMenuActive] = useState(false);
+    const [isButtonPressed, setIsButtonPressed] = useState(0);
+    const items = [
+    {id: 1, value: 'Menu', href: '#!', icon: 'menu'},
+    {id: 2, value: 'Close', href: '#!', icon: 'close'},
+    {id: 3, value: 'Home', href: '#!', icon: 'home'},
+    {id: 4, value: 'Categories', href: '#!', icon: 'categories'},
+    {id: 5, value: 'Subscription', href: '#!', icon: 'subscription'},
+    {id: 6, value: 'My lessons', href: '#!', icon: 'hat_mylesson'},
+    {id: 7, value: 'Watch later', href: '#!', icon: 'watch_later'},
+  ]
+
+  let imageWrapper = !isMenuActive ? 'image__wrapper--active' : '';
+  let svg = isMenuActive ? 'svg--active' : '';
+  let menuText = isMenuActive ? 'menu__text--active' : '';
+
+  return (
+    <div className= 'menu' onClick={() => setIsMenuActive(false)}>
+        <div className={isMenuActive ? 'blur--active' : 'blur'}/>
+        <div className={isMenuActive ? 'menu__content--active' : 'menu__content'} onClick={e => e.stopPropagation()}>
+            <ul className="menu__inner">
+                {
+                    !isMenuActive ? 
+                    <div className='menu__item' onClick={() => setIsMenuActive(!isMenuActive)}>
+                        <div className='image__wrapper image__wrapper--active'>
+                            <div className='svg svg__menu'>
+                                <NavbarSVGSelector icon='menu'/>
+                            </div>
+                        </div>
+                    </div> :
+                    <div className='menu__item' onClick={() => setIsMenuActive(!isMenuActive)}>
+                        <div className='image__wrapper image__wrapper--active'>
+                            <div className='svg'>
+                                <NavbarSVGSelector icon='close'/>
+                            </div>
+                        </div>
+                    </div>
+                }
+                {
```

Comment:
The same comment as previously - you have here long JSX. You can split it into components

Reply:
Let it be optional